### ISSUE TITLE
Request: Add project to nuget

### ISSUE THREAD

**Issue by Dnawrkshp** [mcbride-clint/Blazor.IntroJs#1]
Hey any chance we can get access to this library through nuget?

**Comment by mcbride-clint** [comment]
Yeah definitely. Give me a bit to update the documentation (since I have been putting that off) and I will let you know when its up.

**Comment by mcbride-clint** [comment]
I published an initial version to NuGet as 0.0.1: [https://www.nuget.org/packages/Blazor.IntroJs/](https://www.nuget.org/packages/Blazor.IntroJs/)

This project has been sitting idle for a bit as I have used IntroJs at work and was wanting to learn more about Blazor JsInterop and hopefully be able to utilize it at work in the future.

If you run into any issues feel free to let me know. The small sample WASM app is the only project it has been used in so far.
